dataset_builders/hf/sciarg/README.md

```diff
@@ -0,0 +1,227 @@
+rhetorical dimensions of scientific writing." ([Lauscher et al., 2018](<(https://aclanthology.org/W18-5206/)>), pp. 40-41)
+
+### Supported Tasks and Leaderboards
+
+- **Tasks**: Argumentation Mining, Component Identification, Relation Identification
+- **Leaderboard:** [More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+
+### Languages
+
+The language in the dataset is English (scientific academic publications on computer graphics).
+
+## Dataset Structure
+
+### Data Instances
+
+```
+{
+    'document_id': 'A10',
+    'text': '<?xml version="1.0" encoding="UTF-8" standalone="no"?> <Document...',
+    `text_bound_annotations`: [{
+        'offset': [ [ 2452, 2582 ] ],
+        'text': "['Skinning is an important part of realistic articulated body animation and is an important topic of computer graphics and animation']",
+        'type': "background_claim",
+        'id': 'T1' ,
+    }, {
+        'offset': [ [ 2595, 2684 ] ],
+        'text': "['skinning can be categorized into algorithmic, physically-based, and example-based methods']",
+        'type': "background_claim",
+        'id': 'T2' ,
+    },
+    ...]
+    `relations`: [{
+        'id': "R1",
+        'head': { "ref_id": "T3", "role": "Arg1" },
+        'tail': { "ref_id": "T4", "role": "Arg2" },
+        'type': "contradicts",
+    }, {
+        'id':, "R2"
+        'head': { "ref_id": "T3", "role": "Arg1" },
+        'tail': { "ref_id": "T5", "role": "Arg2" },
+        'type': "supports",
+    },
+    ...]
+}
+```
+
+### Data Fields
+
+- `document_id`: the base file name, e.g. "A28", a `string` feature
+- `text`: the parsed text of the scientific publication in the XML format, a `string` feature
+- `text_bound_annotations`: span annotations that mark argumentative discourse units (ADUs), a `list` of `dictionay` feature. Each entry has the following fields:
+  - `offsets`: the indices span of the text with the (inclusive) start and the (exclusive) end, respectively, a `list` of `int` feature
+  - `text`: the text that belongs to an ADU, a `string` feature
+  - `type`: the label of the ADU, a `string` feature (See [ADU type list](#components))
+  - `id`: the assigned id to the ADU, a `string` feature
+- `relations`: binary relation annotations that mark the argumentative relations that hold between a head and a tail ADU, a `list` of `dictionay` feature. Each entry has the following fields:
+  - `id`: the assigned id to a relation, a `string` feature
+  - `head`: the first element in a relation, consists of:
+    - `ref_id`: the id of the ADU
+    - `role`: the ordinal id of the unit in the argument
+  - `tail`: the second element in a relation, as `head`, consists of the fields: `ref_id` and `role`.
+  - `type`: the label of the relation, a `string` feature (See [relation type list](#relations))
```

Comment:
The whole `Dataset Instances` should go to https://huggingface.co/datasets/DFKI-SLT/brat (that I meant with "Update the HF dataset card, if necessary" in the PR description TODOs). But `Data Fields` needs a bit of adjustment (any occurence of `ADU` needs to be sth like "span annotation").

Reply:
- imported the current BRAT HF dataset card in d722685
- added contents in the section `Data Instances` and `Data Fields` in b5b580d